src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectExtensions.cs

```diff
@@ -16,14 +16,16 @@ public static class OpenIdConnectExtensions
         /// <param name="services"></param>
         /// <returns></returns>
         public static IServiceCollection AddOpenIdConnectAuthentication(this IServiceCollection services)
-            => services.AddOpenIdConnectAuthentication(OpenIdConnectDefaults.AuthenticationScheme, _ => { });
+        {
+            services.AddSingleton<IConfigureOptions<OpenIdConnectOptions>, OpenIdConnectConfigureOptions>();
```

Comment:
Ah, cool then. Other question: what happens if I call `AddOpenIdConnectAuthentication()` twice? (which is likely not a good idea). Will the "configure options" be run twice? Wouldn't using `TryAddSingleton` make sense here?